[2024-05-05T11:01:33.738+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-05T11:01:33.769+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sample.transform_to_db scheduled__2024-05-05T10:00:00+00:00 [queued]>
[2024-05-05T11:01:33.781+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sample.transform_to_db scheduled__2024-05-05T10:00:00+00:00 [queued]>
[2024-05-05T11:01:33.782+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-05T11:01:33.802+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): transform_to_db> on 2024-05-05 10:00:00+00:00
[2024-05-05T11:01:33.826+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=109390) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-05T11:01:33.827+0000] {standard_task_runner.py:63} INFO - Started process 109410 to run task
[2024-05-05T11:01:33.827+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'sample', 'transform_to_db', 'scheduled__2024-05-05T10:00:00+00:00', '--job-id', '235', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp181vcqyw']
[2024-05-05T11:01:33.828+0000] {standard_task_runner.py:91} INFO - Job 235: Subtask transform_to_db
[2024-05-05T11:01:33.913+0000] {task_command.py:426} INFO - Running <TaskInstance: sample.transform_to_db scheduled__2024-05-05T10:00:00+00:00 [running]> on host 5bc5768c5236
[2024-05-05T11:01:34.066+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sample' AIRFLOW_CTX_TASK_ID='transform_to_db' AIRFLOW_CTX_EXECUTION_DATE='2024-05-05T10:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-05T10:00:00+00:00'
[2024-05-05T11:01:34.067+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-05T11:01:34.462+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/my_dag.py:128 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2024-05-05T11:01:43.914+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-05T11:01:43.915+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-05T11:01:43.923+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=sample, task_id=transform_to_db, execution_date=20240505T100000, start_date=20240505T110133, end_date=20240505T110143
[2024-05-05T11:01:43.961+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-05T11:01:43.973+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-05T11:01:43.982+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
